PendingReleaseNotes.md

```diff
@@ -5,6 +5,8 @@
 ## Notable Features
 
 ## Breaking Changes
+- The REST API service has been removed. All cluster configuration is now accomplished through the CRDs or with the Ceph tools in the toolbox. 
```

Comment:
maybe link to an example here so people reading the breaking changes can learn more about it easily?

Reply:
ok, I know you love links!